src/main/java/frc/robot/RobotMap.java

```diff
@@ -32,4 +34,25 @@
         DigitalInput getintakePositionLimitSwitch();
     }
 
+    LiftMap getLiftMap();
+
+    interface LiftMap {
+
+        SendableSpeedController getLiftMotor();
+
+        Encoder getHeightEncoder();
+
+        DigitalInput getTopLimitSwitch();
+
+        DigitalInput getBottomLimitSwitch();
+    }
+
+    MaflipulatorMap getMaflipulatorMap();
+
+    interface MaflipulatorMap {
+
+        SendableSpeedController getFlipMotor();
+
+        Potentiometer getManipulatorPot();
```

Comment:
`getMaflipulatorPot`, Please keep the naming consistent as the manipulator is a separate subsystem. (No this probably isn't the best choice)